searx/static/themes/simple/package.json

```diff
@@ -8,6 +8,8 @@
     "grunt-contrib-jshint": "~3.1.1",
     "grunt-contrib-less": "~3.0.0",
     "grunt-contrib-uglify": "~5.0.1",
+    "grunt-xmlmin": "~0.1.8",
+    "grunt-contrib-htmlmin": "~3.1.0",
```

Comment:
OK I understand .. can we optimize this later?  First I think we should clarify which version of svgo we should use. ATM we have an issue reported when `make node.clean node.env` is build:

    npm WARN deprecated svgo@0.6.6: This SVGO version is no longer supported. Upgrade to v2.x.x.

This issue is caused by grunt-webfont and might be also related to https://github.com/searxng/searxng/pull/488#issuecomment-963606592 / Hint: https://github.com/sapegin/grunt-webfont is not maintained since years, see archived repository at https://github.com/sapegin/grunt-webfont and we have other issues with, see  https://github.com/searxng/searxng/pull/248#issuecomment-894672328